buildSrc/src/main/groovy/openhouse.apps-spark-conventions.gradle

```diff
@@ -0,0 +1,123 @@
+plugins {
```

Comment:
Nit. The name `apps-spark-conventions` doesn't sound right. Should `apps-spark-commons` be better?